AWT `io.quarkus.awt.it.ImageGeometryFontsIT` native integration test failing with "GraalVM for Java 20" dev builds

### Describe the bug

$title

cc @Karm 

### Expected behavior

Tests should pass.

### Actual behavior

Test fails with:

```
[ERROR] Failures: 
[ERROR]   ImageGeometryFontsIT>ImageGeometryFontsTest.testGeometryAndFonts:89 TIFF: Wrong pixel at 80x56. Expected: [46,14,32,255] Actual: [194,59,132,255] ==> expected: <true> but was: <false>
[ERROR]   ImageGeometryFontsIT>ImageGeometryFontsTest.testGeometryAndFonts:89 GIF: Wrong pixel at 80x56. Expected: [2,0,0,0] Actual: [156,0,0,0] ==> expected: <true> but was: <false>
[ERROR]   ImageGeometryFontsIT>ImageGeometryFontsTest.testGeometryAndFonts:89 PNG: Wrong pixel at 80x56. Expected: [46,14,32,255] Actual: [194,59,132,255] ==> expected: <true> but was: <false>
[ERROR]   ImageGeometryFontsIT>ImageGeometryFontsTest.testGeometryAndFonts:89 JPG: Wrong pixel at 80x56. Expected: [73,0,44,0] Actual: [201,53,137,0] ==> expected: <true> but was: <false>
[ERROR]   ImageGeometryFontsIT>ImageGeometryFontsTest.testGeometryAndFonts:89 BMP: Wrong pixel at 80x56. Expected: [46,14,32,0] Actual: [194,59,132,0] ==> expected: <true> but was: <false>
```

See https://github.com/oracle/graal/actions/runs/3898752856/jobs/6658142635#step:9:487

### How to Reproduce?

1. Build GraalVM `master` using Java 20 EA
2. Set GRAALVM_HOME to `<graalvm-repo>/sdk/latest_graalvm_home`
3. ./mvnw -Dnative -pl integration-tests/awt -Dnative.surefire.skip -Dformat.skip -Dno-descriptor-tests -Dlog.level=ALL clean verify -Dquarkus.native.container-build=false

### Output of `uname -a` or `ver`

Linux 6.0.17-300.fc37.x86_64 #1 SMP PREEMPT_DYNAMIC Wed Jan 4 15:58:35 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux

### Output of `java -version`

_No response_

### GraalVM version (if different from Java)

GraalVM for Java 20 - dev build based on 92cfa766d1f059561e6c6a5631d6b082fc9f7765

### Quarkus version or git rev

f1be907e9994569e1e321ebda596bf3e49ac4900

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

_No response_

### Additional information

_No response_

--- a/extensions/awt/deployment/src/main/java/io/quarkus/awt/deployment/AwtProcessor.java
+++ b/extensions/awt/deployment/src/main/java/io/quarkus/awt/deployment/AwtProcessor.java
@@ -143,6 +143,8 @@ class AwtProcessor {
                 "sun.font.FontConfigManager",
                 "sun.font.FontManagerNativeLibrary",
                 "sun.font.FontStrike",
+                // Added for JDK 19+ due to: https://github.com/openjdk/jdk20/commit/9bc023220 calling FontUtilities
+                "sun.font.FontUtilities",
                 "sun.font.FreetypeFontScaler",
                 "sun.font.GlyphLayout",
                 "sun.font.GlyphLayout$EngineRecord",

